torch_ort_inference/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# runtime requirements.
+# This list contains default required packages that will be installed via pip.
+# In an environment where a requirement is already met, the package will not be updated.
+# If a package of a specific version is needed, the package can be installed
+# with version being specified (see Readme.md).
+onnxruntime-openvino>=1.12.0
```

Comment:
Shall we remove `onnxruntime-openvino` from `requirements` since it is an extra now? User should install the openvino package using `pip install torch_ort[openvino]` which will install the onnxruntime-openvino package. 
I guess this can be addressed in a subsequent PR.

Reply:
If onnxruntime-openvino is removed, what will be the user experience if user does `pip install torch-ort-inference` without extra? From our last discussions, expected user experience is to have onnxruntime-openvino as default for now.